Server/ReasnAPI/ReasnAPI/Services/EventService.cs

```diff
@@ -190,7 +180,7 @@ private void RemoveTags(Event eventToDelete, int eventId)
         }
     }
 
-    private void RemoveParameters(Event eventToDelete, int eventId)
+    private void RemoveParametersFromEventCollection(Event eventToDelete, int eventId)

```

Comment:
One time you call it Remove*FromEventColletcion, other yime you call it Detach*FromEvent, can we decide on one? + can we use the same method? What's different in that than this Detach one?

Reply:
i removed this methods and used detach ones with passing needed lists instead